templates/advantage/_table.html

```diff
@@ -1,86 +1,83 @@
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+        </tr>
+        <tr id="fips">
+          <td><a href="/security/certifications#fips">FIPS 140-2 Level 1</a> certified crypto modules</td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+        </tr>
+        <tr id="cce">
+          <td><a href="/security/certifications#common-criteria">Common Criteria EAL2</a></td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+        </tr>
+        <tr id="phone-ticket">
+          <td>Phone and ticket support</td>
+          <td class="u-align--center"></td>
+          <td class="u-align--center">Office hours</td>
+          <td class="u-align--center">24/7</td>
+        </tr>
+        <tr id="openstack">
+          <td>OpenStack</td>
+          <td class="u-align--center">Security updates</td>
+          <td class="u-align--center">Updates + support</td>
+          <td class="u-align--center">Updates + support</td>
+        </tr>
+        <tr id="kubernetes">
+          <td>Kubernetes</td>
+          <td class="u-align--center">Security updates</td>
+          <td class="u-align--center">Updates + support</td>
+          <td class="u-align--center">Updates + support</td>
+        </tr>
+        <tr id="lvm-lxd">
+          <td>KVM/LXD</td>
+          <td class="u-align--center">Security updates</td>
+          <td class="u-align--center">Updates + support</td>
+          <td class="u-align--center">Updates + support</td>
+        </tr>
+        <tr>
+          <td>Price</td>
+          <td class="u-align--center">$9</td>
+          <td class="u-align--center">$99</td>
+          <td class="u-align--center">$999</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <p><a href="https://buy.ubuntu.com" class="js-invoke-modal p-button--positive">Visit our shop</a> to purchase a subscription.</p>
```

Comment:
I don't think this needs the `js-invoke-modal` class, as that's what it currently does, invokes the modal rather than takes me to buy.ubuntu.com